kyaml/yaml/fns_test.go

```diff
@@ -129,28 +129,12 @@ func TestElementSetter(t *testing.T) {
 	node = orig.Copy()
 	// Return error because ElementSetter doesn't support a single key
 	// when there is a scalar value in the list
-	_, err = node.Pipe(ElementSetter{Key: "a"})
+	_, err = node.Pipe(ElementSetter{Keys: []string{"a"}})
 	assert.EqualError(t, err, "wrong Node Kind for  expected: MappingNode was ScalarNode: value: {scalarValue}")
 
-	node = MustParse(`
-- a: b
-- c: d	
-`)
-	// {a: b} is removed since the value is omitted and only key is used
-	// to match and no Element specified.
-	rn, err = node.Pipe(ElementSetter{Key: "a"})
-	assert.NoError(t, err)
-	assert.Nil(t, rn)
-	assert.Equal(t, `- c: d
-`, assertNoErrorString(t)(node.String()))
```

Comment:
Please keep a version of this test as regression coverage,
to record what what happens if a key is specified with no value specified
(as in line 132 in the new code), but with the input is as given here
(which is a list with two entries, and each entry is a map, and each map
has only one entry).

Maybe it just moved?  I looked but didn't see it